pom.xml

```diff
@@ -82,6 +130,23 @@
       </plugin>
 
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
We were doing it before https://github.com/GoogleCloudPlatform/app-tools-lib-for-java/blob/985c162d0dc472d273ace1aff5ea47fcd9a79477/pom.xml#L124